guides/common/modules/proc_renaming-smart-proxy.adoc

```diff
@@ -52,11 +53,11 @@ In this example the archive file is copied to the `root` user’s home directory
 ----
 +
 Ensure that you enter the full path to the `.tar` file.
-. Optional: If you have created a custom certificate for {SmartProxyServer}, on {SmartProxyServer}, to deploy the certificate, enter the `{foreman-installer}` command that the `{certs-generate}` command returns.
+. If you have created a custom certificate for {SmartProxyServer}, deploy the certificate on {SmartProxyServer} by entering the `{foreman-installer}` command that the `{certs-generate}` command returned in a previous step.
```

Comment:
Maybe instead of "Do X by entering the Y command" > Run Y to do X. IMHO no need for "the" + "command".

Reply:
Honestly, I don't see how to do what you suggest without omitting the "that the {certs-generate} command returned" part. And that seems too important to leave out.

Maybe there is a way to phrase it all in a better way. If  you can think of any, feel free to suggest something. Otherwise, I'll keep it as it is, even though I wouldn't exactly call it perfect.